scripts/run_backend_tests.py

```diff
@@ -367,14 +396,22 @@ def check_test_results(tasks, task_to_taskspec, generate_coverage_report):
             )
 
             try:
+                if not tests_failed_regex_match:
+                    # There was an internal error, and the tests did not run
+                    # (The error message did not match
+                    # `tests_failed_regex_match`).
```

Comment:
I think the comment is a bit redundant now we have a message in the exception.

Reply:
Removed the comment